### PR TITLE
Feat/layout style

### PR DESCRIPTION
**Pull request by RMcNeely**
Adds a traditional stacked pancake layout for the main body of the web page. Included a new navbar layout to correctly fit into the grid.

### CODE REVIEW

**Comment by RMcNeely** [comment]
You called me out on my placeholder layout, I was planning on moving to this once I got around to a header and pages but you caught on the mobile layout.

File: src/index.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ title: "ByteZone Dev"
 
 ## Come hire us!
 
-Here at ByteZone we're a developement firm focused on building exceptional solutions so you can focus on your business.
+Here at ByteZone we're a development firm focused on building exceptional solutions so you can focus on your business.
```
Comment (CodeMan99) [inline src/index.md:10]: When rendered, is there now whitespace between this "element" and the next paragraph?

Comment (RMcNeely) [reply src/index.md:10]: No, but then again there isn't any now.

Comment (CodeMan99) [reply src/index.md:10]: Seems like we want that margin. Is this the PR to resolve that in?

Comment (RMcNeely) [reply src/index.md:10]: Yeah, let's add some default spacing. A layout PR seems like the appropriate place for that 👍

Comment (RMcNeely) [reply src/index.md:10]: Check it out now with the stack. That adds spacing between every set of tags in the main element (and anything else that adds that new class)

**Review by CodeMan99 — APPROVED** [review]

**Pull request merged**